x-pack/plugins/fleet/server/services/epm/kibana/assets/install.ts

```diff
@@ -294,6 +294,7 @@ export async function installKibanaSavedObjects({
         overwrite: true,
         readStream: createListStream(toBeSavedObjects),
         createNewCopies: false,
+        refresh: false,
```

Comment:
Added `refresh:false` to saved object importer as discussed [here](https://github.com/elastic/ingest-dev/issues/1468#issuecomment-1373374529).
This speeds up the total install time by about 3 mins (2s/package for the slowest packages).

Reply:
We might want this change in a separate PR so we can revert (if needed) it without reverting the other test improvements